client/src/Components/UserLogin.js

```diff
@@ -1,97 +1,148 @@
+import React, { useState, useEffect } from "react";
+import { Redirect } from 'react-router-dom'
 import Button from '@material-ui/core/Button';
 import CssBaseline from '@material-ui/core/CssBaseline';
 import TextField from '@material-ui/core/TextField';
-import Link from '@material-ui/core/Link';
-import Grid from '@material-ui/core/Grid';
+import Avatar from '@material-ui/core/Avatar';
 import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import Grid from '@material-ui/core/Grid';
 import Typography from '@material-ui/core/Typography';
 import { makeStyles } from '@material-ui/core/styles';
 import Container from '@material-ui/core/Container';
+import Link from '@material-ui/core/Link';
+
+
+function Login(props) {
+  const useStyles = makeStyles(theme => ({
+    paper: {
+      marginTop: theme.spacing(8),
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+    },
+    avatar: {
+      margin: theme.spacing(1),
+      backgroundColor: theme.palette.secondary.main,
+    },
+    form: {
+      width: '100%', 
+      marginTop: theme.spacing(3),
+    },
+    submit: {
+      margin: theme.spacing(3, 0, 2),
+    },
+  }));
+  
+  const classes = useStyles()
 
 
-//User or Org
-// Email
-// pass
-//login button
-const useStyles = makeStyles(theme => ({
-  paper: {
-    marginTop: theme.spacing(8),
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
-  },
-  avatar: {
-    margin: theme.spacing(1),
-    backgroundColor: theme.palette.secondary.main,
-  },
-  form: {
-    width: '100%', // Fix IE 11 issue.
-    marginTop: theme.spacing(1),
-  },
-  submit: {
-    margin: theme.spacing(3, 0, 2),
-  },
-}));
+  // Declare and initialize state variables
+  let [email, setEmail] = useState('')
+  let [message, setMessage] = useState('')
+  let [password, setPassword] = useState('')
 
-export default function Login() {
-  const classes = useStyles();
+  useEffect(() => {
+    setMessage('');
+  }, [email, password])
+
+  // Event handlers
+  const handleSubmit = e => {
+    e.preventDefault()
+    // make a fetch request to get route of the server to check for authentication
+    fetch(`${process.env.REACT_APP_SERVER_URL}/auth/login`, {
+      method: 'POST',
+      body: JSON.stringify({
+        email,
+        password
+      }),
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    })
+    // setMessage to erroe if not authenticate
+      .then(response => {
+        if (!response.ok) {
+          setMessage(`${response.status}: ${response.statusText}`)
+          return;
+        }
+        
+        response.json().then(result => {
+          props.updateUser(result.token);
+        })
+      }).catch(err => {
+        console.log(err);
+        setMessage(`${err.toString()}`)
+    })
+  }
+
+  // if authenticated we are updateUser + redirect to profile
+  if (props.user) {
+    return <Redirect to="/profile" />
```

Comment:
The first path they should redirect to is `/home` because that is the main dashboard with all the information.